lib/Importer/ONNXIFILoader.cpp

```diff
@@ -18,6 +18,8 @@
 
 #include "onnx/onnx.pb.h"
 
+#include <iostream>
```

Comment:
I know that this is WIP, I just want to point out that in most parts of the code we use llvm::outs() or llvm::errs().

Reply:
yup, those std::cout will go away, just using that for debugging. Will change for llvm::outs() for current debugging.